src/View/Cell.php

```diff
@@ -232,12 +232,13 @@ public function render($template = null)
      * @param string $action The action invoked.
      * @return array The cache configuration.
      */
-    protected function _cacheConfig($action)
+    protected function _cacheConfig($action, $template = null)
     {
         if (empty($this->_cache)) {
             return [];
         }
-        $key = 'cell_' . Inflector::underscore(get_class($this)) . '_' . $action;
+        $template = $template ?: "default";
+        $key = 'cell_' . Inflector::underscore(get_class($this)) . '_' . $action . '_' . $template;
```

Comment:
Probably add name to to end of this $var ,make confilict with other generated name